### DISCUSSION TITLE
Cannot delete run log as superadmin

### DISCUSSION THREAD

**Discussion by alfredkzr** [windmill-labs/windmill#4174] [Questions] [answered]
Hello,

I was testing around and noticed that I couldn't delete any run logs (attached screenshot) as superadmin. Am I doing something wrongly?

My instance is running locally via docker-compose, this is not unique to my local machine as I have another instance running on a single host AWS EC2 using docker-compose as well.

<img width="1586" alt="cannot-delete-run-log" src="https://github.com/user-attachments/assets/93ffa8bd-8591-4593-8a17-fee7a9a6c136">

**Comment by rubenfiszel** [comment, marked as answer]
Should be fixed on very latest, thanks for report

**Reply by zross11** [reply]
Is this broken again? I'm getting a `SqlErr: no column found for name: labels @jobs.rs:5810:17` and also run a docker instance (although I am only admin)

**Reply by rubenfiszel** [reply]
it was, fixed on latest again
